source/Irrlicht/OpenGL/Driver.h

```diff
@@ -404,11 +413,11 @@ namespace video
 		void addDummyMaterial(E_MATERIAL_TYPE type);
 
 		unsigned QuadIndexCount;
-		GLuint QuadIndexBuffer = 0;
+		unsigned int QuadIndexBuffer = 0;
 		void initQuadsIndices(int max_vertex_count = 65536);
 
-		void debugCb(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length, const GLchar *message);
-		static void APIENTRY debugCb(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length, const GLchar *message, const void *userParam);
+		void debugCb(unsigned int source, unsigned int type, unsigned int id, unsigned int severity, int length, const char *message);
+		static void APIENTRY debugCb(unsigned int source, unsigned int type, unsigned int id, unsigned int severity, int length, const char *message, const void *userParam);
```

Comment:
changing all these types certainly doesn't seem ideal.

do our gl bindings not provide them? if so that should be fixed instead
see also #168, maybe we should switch to something else?

Reply:
> changing all these types certainly doesn't seem ideal.

And isn’t even required to avoid *linking* to `libGL`. They only need headers. (likewise with constants btw but they are fine when used via `GL.`)